src/components/WalletStatementModal/index.native.js

```diff
@@ -25,15 +26,21 @@ class WalletStatementModal extends React.Component {
      * @param {Object} params
      * @param {String} params.url
```

Comment:
Do we need to update these params up here?

Reply:
Updated!